x-pack/plugins/uptime/public/components/common/header/page_header.tsx

```diff
@@ -44,28 +45,29 @@ export const PageHeader = () => {
   const DatePickerComponent = () =>
     isCertRoute ? (
       <CertRefreshBtn />
-    ) : isStepDetailRoute ? null : (
+    ) : (
       <StyledPicker grow={false} style={{ flexBasis: 485 }}>
         <UptimeDatePicker />
       </StyledPicker>
     );
 
-  const isMonRoute = useRouteMatch(MONITOR_ROUTE);
+  const isMonitorRoute = useRouteMatch(MONITOR_ROUTE);
+
+  if (isStepDetailRoute) {
+    return null;
+  }
 
   return (
     <>
       <SyntheticsCallout />
       <EuiFlexGroup alignItems="center" justifyContent="spaceBetween" wrap responsive={false}>
-        <EuiFlexItem>
-          <PageTabs />
-        </EuiFlexItem>
+        <EuiFlexItem>{isMonitorRoute ? <MonitorPageTitle /> : <PageTabs />}</EuiFlexItem>
```

Comment:
It's a bit weird to me that the header should care about what page it's on. It would make more sense, IMHO, for the page to tell the header what should be in the header, that would, IMHO be less fragile. Each page could just say explicitly "I want this type of header". 

Thinking ahead here, when we add a new page we want whoever works on it to make an explicit choice of header.